metadata-ingestion/docs/sources/openapi/openapi_recipe.yml

```diff
@@ -4,9 +4,15 @@ source:
     name: test_endpoint # this name will appear in DatHub
     url: https://test_endpoint.com/
     swagger_file: classicapi/doc/swagger.json  # where to search for the OpenApi definitions
-    get_token:  # optional, if you need to get an authentication token beforehand
+
+    # optional, if you need to get an authentication token beforehand
+    get_token:
         request_type: get
-        url: api/authentication/login?username={username}&password={password}
+        url_complement: api/authentication/login?username={username}&password={password}
```

Comment:
I think this still can be more clear

can username/password be used without get_token?

can I used both get_token and bearer_token?

ideally this would be like "option 1: bearer token" "option 2: dynamically generated tokens" - more prescriptive on how the configs should be used

Reply:
Okay got it, both are optional. I ll do this change